tailwind.config.js

```diff
@@ -83,6 +98,7 @@ module.exports = {
             boxShadow: {
                 '1': '0px 9px 44px rgba(0, 0, 0, 0.06)',
                 '2': '0px 20px 41px rgba(0, 0, 0, 0.25)',
+                '3': '0px 1px 6px rgba(0, 0, 0, 0.1)',
```

Comment:
No `px` values only use `rem`.

Reply:
These shadows are imported from Figma so the design impact should not be changed.
Tailwind CSS also used shadows in px instead of rem 
![image](https://user-images.githubusercontent.com/5028960/207412661-da4e2479-8f3b-48bc-98af-a4f8648ac7cb.png)

Reply:
that's fine, please use this https://nekocalc.com/px-to-rem-converter to convert